reddit_decider/__init__.py

```diff
@@ -808,6 +762,39 @@ def get_all_dynamic_configs(self) -> List[Dict[str, Any]]:
 
         return parsed_configs
 
+    def _get_decision(
+        self,
+        experiment_name: str,
+        ctx: Dict[str, Any],
+    ) -> Optional[Decision]:
+        if self._internal is None:
+            logger.error("RustDecider is None--did not initialize.")
+            return None
+
+        try:
+            return self._internal.choose(experiment_name, ctx)
+        except FeatureNotFoundException as exc:
+            warnings.warn(str(exc))
```

Comment:
🔕 what in the world is a "warning" in this context??

Reply:
ah, this maintains parity with this og pr: https://github.com/reddit/experiments.py/pull/2

it only logs "Feature "{feature_name}" not found." once via `warnings.warn()` to alleviate logs if the feature was disabled and not found in config. 

"Repetitions of a particular warning for the same source location are typically suppressed." - [docs](https://docs.python.org/3/library/warnings.html)